chain/message_store.go

```diff
@@ -13,14 +13,14 @@ import (
 // MessageProvider is an interface exposing the load methods of the
 // MessageStore.
 type MessageProvider interface {
-	LoadMessages(context.Context, cid.Cid) ([]*types.SignedMessage, error)
+	LoadMessages(context.Context, types.TxMeta) ([]*types.SignedMessage, []*types.SignedMessage, error)
```

Comment:
Please name these return values

Reply:
It might be too late, but since the TxMeta and the execution order of these messages is defined to be BLS and then SECP, returning them in that order might slightly reduce cognitive friction. Maybe this will be more obvious when the BLS ones are not `SignedMessage`, which I assume is coming in a follow-up.

Reply:
This is definitely covered in the next PR.